front-end/src/pages/AuthorPage.tsx

```diff
@@ -63,69 +62,67 @@ export default function AuthorPage(props: any) {
           setIsFollower(false);
         });
       }
-
-      AxiosWrapper.get(authorUrl + "/posts/", props.loggedInUser).then((res: any) => {
-        const posts: Post[] = res.data;
-        setPostEntries(posts);
-      }).catch((err: any) => {
-        console.error("ERROR GETTING POSTS");
-        setResponseMessage(500);
-      })
     };
   }, []);
 
+  // Author's profile pic will be the same one from their GitHub
+  const profilePic = (github: string | undefined) => {
+    if (github && github.split('https://github.com/').pop()) {
+      return <CardImg top width="100%" src={github + ".png"} alt="Card image cap" />
+    } else {
+      return <CardBody>{Icons.defaultProfilePic}</CardBody>;
+    }
+  }
+
   if (responseMessage > 299) {
     return (<Container>
       <Alert color="danger">An error occurred! Please try again</Alert>
     </Container>)
   }
 
-  const displayFollowButton = () => {
-    if (props.loggedInUser && author?.id !== props.loggedInUser.authorId) {
-      return <FollowRequestButton loggedInUser={props.loggedInUser} currentAuthor={author} isFollower={isFollower} setIsFollower={setIsFollower} />
-    }
-  }
 
-  const displayFollowListButtons = () => {
+  const displayProfileButtons = () => {
     if (props.loggedInUser && author?.id === props.loggedInUser.authorId) {
       return (<>
-        <CardText>
-          <Link className="text-white" to={{ pathname: `/author/${props.loggedInUser.authorId}/followers` }}><Button>Followers</Button></Link>
-        </CardText>
-        <CardText>
-          <Link className="text-white" to={{ pathname: `/author/${props.loggedInUser.authorId}/following` }}><Button>Following</Button></Link>
-        </CardText>
+        <ListGroupItem tag="a" href={`/author/${props.loggedInUser.authorId}/followers`}>Followers</ListGroupItem>
+        <ListGroupItem tag="a" href={`/author/${props.loggedInUser.authorId}/following`}>Following</ListGroupItem>
       </>)
+    } else {
+      return <FollowRequestButton loggedInUser={props.loggedInUser} currentAuthor={author} isFollower={isFollower} setIsFollower={setIsFollower} />
```

Comment:
I don't think this will properly show up for a foreign author because the "authorUrl" variable from above always uses our back-end URL. Can you put a TODO for this? I can do it in a later PR.

Reply:
Just to clarify, the authorUrl variable always uses our back-end, making it so that the `author` object used here would break if it's not from our back-end.